src/Authentication/Authentication/custom/common/datasample.json

```diff
@@ -0,0 +1,5424 @@
+      "value": "PrintJob.ReadWriteBasic",
+      "adminConsentDescription": "Allows the application to read and update the metadata of print jobs that the signed-in user created. Does not allow access to print job document content.",
+      "type": "User",
+      "adminConsentDisplayName": "Read and write basic information of user's print jobs",
+      "isEnabled": true,
+      "id": "6f2d22f2-1cb6-412c-a17c-3336817eaa82"
+    },
+    {
+      "userConsentDescription": "Allows the application to read and update the metadata of print jobs on your behalf. Does not allow access to print job document content.",
+      "userConsentDisplayName": "Read and write basic information of print jobs",
+      "value": "PrintJob.ReadWriteBasic.All",
+      "adminConsentDescription": "Allows the application to read and update the metadata of print jobs on behalf of the signed-in user. Does not allow access to print job document content.",
+      "type": "Admin",
+      "adminConsentDisplayName": "Read and write basic information of print jobs",
+      "isEnabled": true,
+      "id": "3a0db2f6-0d2a-4c19-971b-49109b19ad3d"
+    },
+    {
+      "userConsentDescription": "Allows the app to read and write your organization's device configuration policies on your behalf.  For example, device registration policy can limit initial provisioning controls using quota restrictions, additional authentication and authorization checks.",
+      "userConsentDisplayName": "Read and write your organization's device configuration policies",
+      "value": "Policy.ReadWrite.DeviceConfiguration",
+      "adminConsentDescription": "Allows the app to read and write your organization's device configuration policies on behalf of the signed-in user.  For example, device registration policy can limit initial provisioning controls using quota restrictions, additional authentication and authorization checks.",
+      "type": "Admin",
+      "adminConsentDisplayName": "Read and write your organization's device configuration policies",
+      "isEnabled": true,
+      "id": "40b534c3-9552-4550-901b-23879c90bcf9"
+    },
+    {
+      "userConsentDescription": "Allows the app to submit application packages to the catalog and cancel submissions that are pending review on your behalf.",
+      "userConsentDisplayName": "Submit application packages to your organization's catalog and cancel pending submissions",
+      "value": "AppCatalog.Submit",
+      "adminConsentDescription": "Allows the app to submit application packages to the catalog and cancel submissions that are pending review on behalf of the signed-in user.",
+      "type": "User",
+      "adminConsentDisplayName": "Submit application packages to the catalog and cancel pending submissions",
+      "isEnabled": true,
+      "id": "3db89e36-7fa6-4012-b281-85f3d9d9fd2e"
+    },
+    {
+      "userConsentDescription": "Allows the app to read the Teams apps that are install
```

Comment:
Let's change this to a random guid, i.e. use `new-guid | select -ExpandProperty guid | clip` to add a guid to your clipboard and paste it in. Basically any id's or names that aren't related to permissions should be replaced to avoid exporting private data from an organization.